services/embeddings/example.py

```diff
@@ -0,0 +1,32 @@
+import os
+from dotenv import load_dotenv
+from embeddings.embeddings import VectorStore
+from embeddings.utils import load_json
+
+def connect():
+    """Initialise the vector store instance to be used by the embeddings_demo service."""
+
+    load_dotenv()
+
+    store = VectorStore(
+        collection_name="demo_project",
+        vectorstore_type="zilliz",
+        embedding_type="openai",
+        connection_args = {
+            "uri": os.getenv('ZILLIZ_CLOUD_URI'),
```

Comment:
Oh @hanna-paasivirta please make sure this uses the same env var as the basic search service :pray: